schema_utils.py

```diff
@@ -229,47 +264,120 @@ def get(cls, validator_id):
     def has_length_at_least(obj, min_value):
         """Returns True iff the given object (a list) has at least
         `min_value` elements.
+
+        Args:
+            obj: *. Object to compare with min_value.
```

Comment:
What do you mean by "comparing the object with min_value", here and below? Shouldn't the length be compared, not the object itself?

Ditto below.